src/actions/analytics.ts

```diff
@@ -71,7 +71,8 @@ export const trackScreenChange = (
 ) => {
   const { analytics } = getState();
   const { [ANALYTICS_MCID]: mcid } = analytics;
-
+  // If in debug mode, set user property 'debug' to true, if prod set to false
+  FBAnalytics().setUserProperties({ debug: __DEV__ ? 'true' : 'false' });
```

Comment:
Does ```__DEV__``` do what I think it does? Basically, we want the debug value to be ```true``` for all nonproduction builds.

Reply:
Also once the user property value is set, on the GTM side it only updates if the value changes.

Reply:
Ya it should. Testflight builds will be false which is expected.